### PR TITLE
Run extended tests with origin built installed release

### LINKED ISSUE #1442: openshift-ansible release fails if tag already exists
Origin release with ecosystem tests fail if tag for openshift-ansible already exists:

See https://ci.openshift.redhat.com/jenkins/job/test_branch_origin_extended_image_registry/232/

```
Creating output directory: /tmp/tito
Tagging new version of openshift-ansible: 3.10.0-0.63.0 -> 3.10.1-1
Traceback (most recent call last):
  File "/usr/bin/tito", line 23, in <module>
    CLI().main(sys.argv[1:])
  File "/usr/lib/python2.7/site-packages/tito/cli.py", line 203, in main
    return module.main(argv)
  File "/usr/lib/python2.7/site-packages/tito/cli.py", line 671, in main
    return tagger.run(self.options)
  File "/usr/lib/python2.7/site-packages/tito/tagger/main.py", line 114, in run
    self._tag_release()
  File "/usr/lib/python2.7/site-packages/tito/tagger/main.py", line 136, in _tag_release
    self._check_tag_does_not_exist(self._get_new_tag(new_version))
  File "/usr/lib/python2.7/site-packages/tito/tagger/main.py", line 502, in _check_tag_does_not_exist
    raise Exception("Tag %s already exists!" % new_tag)
Exception: Tag openshift-ansible-3.10.1-1 already exists!
++ export status=FAILURE
++ status=FAILURE
+ set +o xtrace
########## FINISHED STAGE: FAILURE: BUILD AN OPENSHIFT-ANSIBLE RELEASE [00h 00m 02s] ##########
```

Note - the errant git tag `3.10.1-1` does not exist in the openshift-ansible repo on Github.

### PR DESCRIPTION
**Pull request by adambkaplan**
Fixes #1442

### CODE REVIEW

File: sjb/config/common/test_cases/origin_installed_release.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ parent: 'common/test_cases/origin_release_with_ecosystem.yml'
 overrides:
   provision:
     os: "rhel"
-    stage: "build"
+    stage: "base"
```
Comment (bparees) [inline sjb/config/common/test_cases/origin_installed_release.yml]: @adambkaplan i thought the whole point of us changing these jobs was we were moving away from using a common job that no one else depended on, how are we still in this situation?

Comment (adambkaplan) [reply sjb/config/common/test_cases/origin_installed_release.yml]: this was per @stevekuznetsov - this parameter dictates which AMI is provisioned to start the test.
are you saying we should avoid the override (which I think is possible).

Comment (bparees) [reply sjb/config/common/test_cases/origin_installed_release.yml]: i think i'm asking, "is anyone else using origin_installed_release"?  If so, why aren't they hitting this.  And if not, why are we using it?

Comment (adambkaplan) [reply sjb/config/common/test_cases/origin_installed_release.yml]: @bparees to answer your questions:
1. No, no other tests outside our DevEx extended tests use this
2. Looks like other tests use origin_built_installed_release.yaml as the base - testing now to see if this works (scripts are almost identical).

/hold

**Comment by adambkaplan** [comment]
@bparees refactored tests to use the `origin_built_installed_release` base, which is shared with:

- Extended conformance tests
- Aggregated logging tests
- Ansible logging

**Comment by bparees** [comment]
lgtm, @stevekuznetsov please ack at your convenience.

**Comment by stevekuznetsov** [comment]
Nobody is using that other base now? Seems like a reasonable change to me.

**Comment by stevekuznetsov** [comment]
@bparees @adambkaplan let me know when to merge and you want to roll it out and monitor jobs

**Comment by stevekuznetsov** [comment]
```
$ sjb/push-update-automatic.sh 
[INFO] Checking for existence of test_branch_jenkins_client_plugin...
[INFO] Checking for existence of test_branch_jenkins_openshift_login_plugin...
[INFO] Checking for existence of test_branch_jenkins_plugin...
[INFO] Checking for existence of test_branch_jenkins_sync_plugin...
[INFO] Checking for existence of test_branch_origin_extended_image_ecosystem...
[INFO] Checking for existence of test_pull_request_jenkins_plugin...
[INFO] Checking for existence of test_pull_request_jenkins_sync_plugin...
[INFO] Checking for existence of test_branch_origin_extended_builds...
[INFO] Checking for existence of test_branch_origin_extended_image_registry...
[INFO] Checking for existence of test_pull_request_jenkins_openshift_login_plugin...
[INFO] Checking for existence of test_pull_request_jenkins_client_plugin...
[INFO] Updating test_branch_jenkins_openshift_login_plugin...
[INFO] Updating test_branch_origin_extended_image_ecosystem...
[INFO] Updating test_branch_jenkins_sync_plugin...
[INFO] Updating test_branch_origin_extended_builds...
[INFO] Updating test_branch_origin_extended_image_registry...
[INFO] Updating test_pull_request_jenkins_openshift_login_plugin...
[INFO] Updating test_pull_request_jenkins_client_plugin...
[INFO] Updating test_pull_request_jenkins_plugin...
[INFO] Updating test_branch_jenkins_client_plugin...
[INFO] Updating test_branch_jenkins_plugin...
[INFO] Updating test_pull_request_jenkins_sync_plugin...
```

**Comment by bparees** [comment]
kicked off new runs of our extended jobs.

**Pull request merged**